ui/src/app/shared/directives/autosize.directive.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ * Copyright Alex Shevchenko.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Directive, ElementRef, HostBinding, HostListener, Input, AfterViewInit, Optional, Self } from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+
+/**
+ * Directive to automatically resize a textarea to fit its content.
+ */
+@Directive({
+  selector: 'textarea[textareaAutosize],' +
+  'textarea[autosize]',
```

Comment:
Maybe change to `dataflowAutoResize`? (I think we should prefix all Dataflow directives with something dataflow-specific)